### ISSUE TITLE
is it possible to outsource the @bodyParam to a file like we can do this with @responseFile ?

### ISSUE THREAD

**Issue by michabbb** [knuckleswtf/scribe#794]
### Scribe version

4.29.0

### Your question

thanks

### Docs

- [X] I've checked [the docs](https://scribe.knuckles.wtf/laravel), the [troubleshooting guide](https://scribe.knuckles.wtf/laravel/troubleshooting), and [existing issues](https://github.com/knuckleswtf/scribe/issues?q=is%3Aissue+), but I didn't find a solution

**Comment by shalvah** [comment]
No. However, you can create your own custom tag that does this. https://scribe.knuckles.wtf/laravel/advanced/plugins
